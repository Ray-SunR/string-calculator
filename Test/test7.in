fa
i b a
lb
wb
ea
eb

fa
m b 2 a
lb
wb
eb
ea

fa
n b a
lb
wb
eb
ea

fa
p b a 2
lb
wb
eb
ea

fa
s b a a
lb
wb
ea
eb

fa
t b a abcde
lb
wb
ea
eb

q
