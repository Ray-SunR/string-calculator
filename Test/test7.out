0

0

1
a
0

0

5
abcde
